logic.js

```diff
@@ -3,7 +3,17 @@ const form = document.querySelector('.form');
 const [title, author] = form.elements;
 
 const inputBook = {};
-let books = new Array([]);
+let books;
+books = [{
+  title: 'Das Kapital',
+  author: 'Karl Marx'
+}, {
+  title: '1984',
+  author: 'George Orwell'
+}, {
+  title: 'The desing of everydat things',
+  author: 'Don Norman'
+}];
 
```

Comment:
- [ ] It would be great if we can implement the luxon library as it is one of the requirements. Please check out the next link for more information [luxon library](https://moment.github.io/luxon/#/install)